api/services/AuthorizationPolicy/index.js

```diff
@@ -117,7 +118,8 @@ policy.include('calendarEventInvite', (p) => {
     ({ viewer, entity: { event, status }, action }) =>
       isOrganiser(event, viewer) ||
       isAdmin(viewer) ||
-      (isUser(viewer) && status === 'Requested' && action === ':create'),
+      (isUser(viewer) && status === 'Requested' && action === ':create') ||
```

Comment:
Explain this... I don't think the new check is required, only need to modify the 3rd check. 

Help me understand.

Reply:
I can't understand the use of 3rd condition.
So I used another condition..

If I add event.auto_accept_request  by using or ,It didn't work.

Can you explain the use of third condition?

Reply:
The third condition says that if the action is create and the user is logged in and the status for which the person wants the invite is 'Requested' then allow him. 

I think very check add a required condition for the event.is_requestable to be true.